diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -278,6 +280,67 @@ int DifferentialFunctionComparator::cmpOperations(
                      dyn_cast<CallInst>(R)->getCalledValue())->getName().
                      startswith("simpll"))
                 ModComparator->tryInline = {nullptr, dyn_cast<CallInst>(R)};
+
+            // The equivalent of the case above (function changed to a macro)
+            // can also happen in a way that will result in only one function
+            // call (for example when the macro does not start with a call
+            // instruction).
```

Comment:
Since this is an equivalent of the above, isn't it possible to make a function from this code? The two pieces are very similar.

Reply:
At first I thought the code is quite different, but after looking at it closely it's clear that the code for the case where both instructions are CallInsts is universal for all cases, therefore I moved it into a function and used it in the second two cases.